src/proof/clausal_bitvector_proof.h

```diff
@@ -61,9 +61,16 @@ class ClausalBitVectorProof : public BitVectorProof
 
  protected:
   // A list of all clauses and their ids which are passed into the SAT solver
-  std::vector<std::pair<ClauseId, prop::SatClause>> d_usedClauses;
+  std::map<ClauseId, prop::SatClause> d_clauses;
+  std::vector<ClauseId> d_originalClauseIndices{};
```

Comment:
If we use default member initializers, maybe we should apply them to all members (at least the map)?

Reply:
Yeah, good idea.